WooCommerce/Classes/POS/ViewModels/TotalsViewModel.swift

```diff
@@ -23,16 +22,19 @@ final class TotalsViewModel: ObservableObject {
 
     /// Order created the first time the checkout is shown for a given transaction.
     /// If the merchant goes back to the product selection screen and makes changes, this should be updated when they return to the checkout.
-    @Published private(set) var order: POSOrder?
+    @Published private(set) var order: Order? = nil
+    private var totalsCalculator: OrderTotalsCalculator? = nil
+
     @Published private(set) var isSyncingOrder: Bool = false
 
     @Published private(set) var paymentState: PaymentState = .acceptingCard
 
     @Published private(set) var connectionStatus: CardReaderConnectionStatus = .disconnected
 
     // MARK: - Order total amounts
-
-    @Published private(set) var formattedCartTotalPrice: String?
+    var formattedCartTotalPrice: String? {
+        formattedPrice(totalsCalculator?.itemsTotal.stringValue, currency: order?.currency)
+    }
 
     var formattedOrderTotalPrice: String? {
         formattedPrice(order?.total, currency: order?.currency)
```

Comment:
Question: Just wondering, in `formattedOrderTotalPrice` and `formattedOrderTotalTaxPrice` should we log if there's an error if the order happens to be `nil`? Eg:
```
    var formattedOrderTotalPrice: String? {
        guard let total = order?.total, let currency = order?.currency else {
            DDLogError("Failed to ... ")
            return nil
        }
        return formattedPrice(total, currency: currency)
    }
```

(I'm not sure how expected are those to be nil while the order stuff happens, so maybe it's an overkill and not necessary 🤔 )

Reply:
If the order is still nil after syncing/updating then we will probably get one of the errors that we will handle with the unhappy path error views.
And since the order is initally nil we would log this once for sure and maybe have unnecessary error logs..